lib/vue-lib/src/design-system/menus/DropdownMenuItem.vue

```diff
@@ -69,6 +70,7 @@
       </div>
     </div>
     <div
+      v-if="!centerHeader && !header"
```

Comment:
ah I just realized this conditional isn't quite right - it should be `!(centerHeader && header)`

Reply:
my mistake - fix this and everything else looks great!

Reply:
Done!